src/main/java/org/spongepowered/api/entity/ai/task/AITask.java

```diff
@@ -75,7 +75,7 @@
      * @param other The other task
      * @return True if it can be, false if not
      */
-    boolean canRunConcurrentWith(AITask<O> other);
+    boolean canRunConcurrentWith(AITask<?> other);
```

Comment:
Here `O` is the type of `Agent`. The `other` parameter's generic could either be `O`'s subclass or superclass. For example, if this task applies to `Monster`s, the `other` task could apply to either `Agent`s and `Spider`, if both tasks exists on the goal of a spider.